app/code/Magento/Downloadable/Model/Link/Builder.php

```diff
@@ -132,6 +132,10 @@ public function build(\Magento\Downloadable\Api\Data\LinkInterface $link)
         if (isset($this->data['is_unlimited']) && $this->data['is_unlimited']) {
             $link->setNumberOfDownloads(0);
         }
+
+        if (isset($this->data['use_default_title']) && $this->data['use_default_title'] == '1') {
```

Comment:
What do you think about applying the following minor adjustment?
```suggestion
        $useDefaultTitle = $this->data['use_default_title'] ?? false;
        
        if ($useDefaultTitle) {
```

Reply:
Thanks for your idea.
I will apply your solution, it makes my code neater